.github/workflows/integration-test.yml

```
@@ -336,13 +336,13 @@ jobs:
         id: get-matrix
         run: |
           includes=""
-          for service in "mongodb" "kafkametrics"; do
+          for service in "mongodb" "kafkametrics" "jmx/cassandra"; do
             for arch in "amd64" "arm64"; do
               if [ "$service" = "mongodb" ]; then
                 for mongodb_version in "4.0" "4.4" "5.0" "6.0" "7.0"; do
                   includes="${includes},{\"SERVICE\": \"${service}\", \"ARCH\": \"${arch}\", \"MONGODB_VERSION\": \"${mongodb_version}\"}"
                 done
-              else
+              elif [ "$service" != "jmx/cassandra" && "arm64" != "$arch" ]; then
```

Need double brackets:
```suggestion
              elif [[ "$service" != "jmx/cassandra" && "arm64" != "$arch" ]]; then
```